internal/xds/resolver/xds_resolver_test.go

```diff
@@ -422,30 +428,32 @@ func (s) TestResolverGoodServiceUpdate(t *testing.T) {
 				ClusterSpecifierType: e2e.RouteConfigClusterSpecifierTypeWeightedCluster,
 				WeightedClusters:     map[string]int{"cluster_1": 75, "cluster_2": 25},
 			}),
+			clusterConfig:  []*v3clusterpb.Cluster{e2e.DefaultCluster("cluster_1", "endpoint_1", e2e.SecurityLevelNone), e2e.DefaultCluster("cluster_2", "endpoint_2", e2e.SecurityLevelNone)},
+			endpointConfig: []*v3endpointpb.ClusterLoadAssignment{e2e.DefaultEndpoint("endpoint_1", defaultTestHostname, defaultTestPort), e2e.DefaultEndpoint("endpoint_2", defaultTestHostname, defaultTestPort)},
 			// This update contains the cluster from the previous update as well
 			// as this update, as the previous config selector still references
 			// the old cluster when the new one is pushed.
 			wantServiceConfig: `{
-  "loadBalancingConfig": [{
-    "xds_cluster_manager_experimental": {
-      "children": {
-        "cluster:cluster_1": {
-          "childPolicy": [{
-			"cds_experimental": {
-			  "cluster": "cluster_1"
-			}
-		  }]
-        },
-        "cluster:cluster_2": {
-          "childPolicy": [{
-			"cds_experimental": {
-			  "cluster": "cluster_2"
-			}
-		  }]
-        }
-      }
-    }
-  }]}`,
+		  "loadBalancingConfig": [{
+		    "xds_cluster_manager_experimental": {
+		      "children": {
+		        "cluster:cluster_1": {
+		          "childPolicy": [{
+					"cds_experimental": {
+					  "cluster": "cluster_1"
+					}
+				  }]
+		        },
+		        "cluster:cluster_2": {
+		          "childPolicy": [{
+					"cds_experimental": {
+					  "cluster": "cluster_2"
+					}
+				  }]
+		        }
+		      }
+		    }
+		  }]}`,
```

Comment:
Can we undo this diff to keep the changes in this PR limited to the main intent of the PR. You could send a separate PR that *only* focuses on whitespace changes. Thanks.